markwon/src/main/java/ru/noties/markwon/renderer/SpannableMarkdownVisitor.java

```diff
@@ -465,23 +481,13 @@ public void visit(Link link) {
         setSpan(length, factory.link(theme, destination, configuration.linkResolver()));
     }
 
-    private void setSpan(int start, @Nullable Object span) {
-        if (span != null) {
-
-            final int length = builder.length();
-
-            SpannableBuilder.setSpans(
-                    builder,
-                    span,
-                    start,
-                    length
-            );
-        }
+    private void setSpan(int start, @Nullable Object spans) {
+        builder.setSpan(spans, start, builder.length(), Spanned.SPAN_EXCLUSIVE_EXCLUSIVE);
```

Comment:
I think we should check for `null` here. And SpannableBuilder must not allow passing nulls

Reply:
Sure but why? What is your argument for not allowing `SpannableBuilder` to accept `@Nullable`?

Reply:
I don't know, it feels wrong. We _could_ but only if we introduce own method for applying array of spans. And it should have an explicit annotation